### PR TITLE
Added form configure options fixer

### PR DESCRIPTION
**Pull request by umpirsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ $ symfony-upgrade-fixer fix /path/to/code --dry-run
 
 | Name  | Description |
 | ----  | ----------- |
+| form_configure_options | The method AbstractType::setDefaultOptions(OptionsResolverInterface $resolver) have been renamed to AbstractType::configureOptions(OptionsResolver $resolver). |
```
Comment (akovalyov) [inline README.md]: has been renamed

File: README.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ $ symfony-upgrade-fixer fix /path/to/code --dry-run
...
 | form_events | The events PRE_BIND, BIND and POST_BIND were renamed to PRE_SUBMIT, SUBMIT and POST_SUBMIT. |
 | form_option_names | Options precision and virtual was renamed to scale and inherit_data. |
```
Comment (akovalyov) [inline README.md]: were renamed

File: src/Symfony/Upgrade/Fixer/FormConfigureOptionsFixer.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            $this->fixConfigureOptions($tokens);
+            $this->fixUseStatements($tokens);
+        }
+
+        return $tokens->generateCode();
+    }
+
+    public function getDescription()
+    {
+        return 'The method AbstractType::setDefaultOptions(OptionsResolverInterface $resolver) have been renamed to AbstractType::configureOptions(OptionsResolver $resolver).';
```
Comment (akovalyov) [inline src/Symfony/Upgrade/Fixer/FormConfigureOptionsFixer.php]: has been renamed :)

File: README.md (hunk 2 above)
Comment (umpirsky) [inline README.md]: You think grammar nazzi like me didn't notice. :)

https://github.com/symfony/symfony/blob/2.8/UPGRADE-3.0.md is full of inconsistency, I just followed.

**Comment by borNfreee** [comment]
:+1:

**Pull request merged**